plugin.py

```diff
@@ -16,7 +16,19 @@ class LspVuePlugin(NpmClientHandler):
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', 'vue-language-server', 'bin', 'vls')
 
-    def on_client_configuration_ready(self, configuration: dict):
+    @classmethod
+    def configuration(cls) -> sublime.Settings:
+        settings = super().configuration()
+        options = settings.get('initializationOptions', {})
+        cls._add_formatting_options(options)
+        settings.set('initializationOptions', options)
+        return settings
+
+    def on_client_configuration_ready(cls, configuration: dict):
```

Comment:
For clarity should be annotated with `@classmethod`.

Reply:
I got a bit confused here. In LSP-vue this is not a classmethod, but in NpmClientHandler it is a classmethod?

Reply:
I think the annotation was missing but it probably worked the same way regardless.